src/js/components/Calendar/stories/Simple.js

```diff
@@ -1,8 +1,16 @@
 import React, { Component } from 'react';
 import { storiesOf } from '@storybook/react';
+import { deepMerge } from 'grommet/utils';
+
+import { Box, Calendar, Grommet, grommet } from 'grommet';
+
+const customTheme = deepMerge(grommet,{
+  calendar:{
+    heading: {
+      level: '3' },
```

Comment:
Something with your braces on the last couple PR doesn't look right, you should handle the auto correctness of your local prettier or configuration.